src/overlays/actors/ovl_Obj_Dowsing/z_obj_dowsing.c

```diff
@@ -10,11 +10,12 @@
 
 #define THIS ((ObjDowsing*)thisx)
 
+s32 ObjDowsing_GetFlag(ObjDowsing* this, GlobalContext* globalCtx);
+s32 func_80B23DD0(ObjDowsing* this, GlobalContext* globalCtx);
 void ObjDowsing_Init(Actor* thisx, GlobalContext* globalCtx);
 void ObjDowsing_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void ObjDowsing_Update(Actor* thisx, GlobalContext* globalCtx);
```

Comment:
```suggestion
void ObjDowsing_Init(Actor* thisx, GlobalContext* globalCtx);
void ObjDowsing_Destroy(Actor* thisx, GlobalContext* globalCtx);
void ObjDowsing_Update(Actor* thisx, GlobalContext* globalCtx);

s32 ObjDowsing_GetFlag(ObjDowsing* this, GlobalContext* globalCtx);
s32 func_80B23DD0(ObjDowsing* this, GlobalContext* globalCtx);
```
We always put the other functions in the actor *below* the external ones.